### PR TITLE
Allow skip messages and add to all tests.

### PR DESCRIPTION
**Pull request by arscan**
I also fixed a couple of minor bugs on our side... like 200's are allowed on deletion as long as you return content in the body of the response.

### CODE REVIEW

File: lib/tests/suites/connectathon_patch_track.rb [hunk 1]
```diff
@@ -96,7 +96,7 @@ def teardown
             validates resource: 'MedicationRequest', methods: ['read']
           }
 
-          skip if @previous_version_id.nil?
+          skip "VersionId of Existing Medicatation Request not returned in C12PATCH_1_(#{fmt})." if @previous_version_id.nil?
```
Comment (jawalonoski) [inline lib/tests/suites/connectathon_patch_track.rb]: spelling error: "Medicatation"

File: lib/tests/suites/resource_test.rb [hunk 2]
```diff
@@ -675,9 +675,18 @@ def teardown
           result.update(STATUS[:skip],"Preexisting #{resource_class.name.demodulize} unknown.", nil)
         else
           reply = @client.destroy(@resource_class,@preexisting_id)
-          if reply.code==204
+          if reply.code==204 and reply.body.blank?
```
Comment (jawalonoski) [inline lib/tests/suites/resource_test.rb]: Don't use `and` use `&&`

File: lib/tests/suites/connectathon_patch_track.rb (hunk 1 above)
Comment (arscan) [reply lib/tests/suites/connectathon_patch_track.rb]: Thanks, I'll fix.

File: lib/tests/suites/resource_test.rb (hunk 2 above)
Comment (arscan) [reply lib/tests/suites/resource_test.rb]: ok, i'll fix

Comment (jawalonoski) [reply lib/tests/suites/resource_test.rb]: There were a few of these... I didn't mark them all, so make sure you change all of them.

**Comment by arscan** [comment]
All set on those comments I think, thanks for reviewing.

**Pull request merged**